lib/inventory_refresh/inventory_collection/helpers/initialize_helper.rb

```diff
@@ -177,9 +177,12 @@ def init_ic_relations(dependency_attributes, parent_inventory_collections = nil)
         #        for the batch saver strategy.
         # @param targeted [Boolean] True if the collection is targeted, in that case it will be leveraging :manager_uuids
         #        :parent_inventory_collections and :targeted_arel to save a subgraph of a data.
+        # @param track_record_changes [Boolean/Array] By default false.  If false no changes to the InventoryObject will be stored,
+        #        otherwise changes to properties enumerated in the array will be stored for use by other inventory collections
+        #        Example: :track_record_changes => [:name, :raw_power_state]
```

Comment:
NOTE: I went back and forth a lot on a separate attribute to track the filter plus a boolean to turn on/off, versus combining them all in one attribute.  On one hand it is handy to only need one attribute (we have a lot of knobs here already) on the other hand could be confusing to have a boolean/array datatype.